### PR TITLE
Added ubid in building_sync.py for seed import

### PR DESCRIPTION
**Pull request by lainsworth8801**
Added ubid @building level in building_sync.py as a required element for seed to import.

Screen Shot 2019-10-29 at 5 02 53 PM

#1740

### CODE REVIEW

**Comment by nllong** [comment]
@lainsworth8801 -- did you commit any of the test files that have UBIDs? I looked int he files under seed/building_sync/tests/data and I couldn't find any UBIDs in those XMLs.

File: seed/building_sync/building_sync.py [hunk 1]
```diff
@@ -90,6 +90,11 @@ class BuildingSync(object):
                 "required": True,
                 "type": "string",
             },
+            "ubid": {
+                "path": "auc:Sites.auc:Site.auc:Buildings.auc:Building.auc:Ubid",
```
Comment (nllong) [inline seed/building_sync/building_sync.py]: According to the schema, the UBID is under Identifier Label (//element(*,auc:SiteType)/auc:Buildings/auc:Building/auc:PremisesIdentifiers/auc:PremisesIdentifier/auc:IdentifierLabel):

image

Which would make this looks like these other mapping entries:

image

Comment (lainsworth8801) [reply seed/building_sync/building_sync.py]: Don't believe i did. There isn't any if i remember right but I modified one locally for testing.

**Comment by nllong** [comment]
Okay. Can you add a UBID to some of the test BuildingSync XML files so that it is tested correctly?

Da: Lin Ainsworth <notifications@github.com>
Risposta: SEED-platform/seed <reply@reply.github.com>
Data: lunedì novembre 4 2019 09:05
A: SEED-platform/seed <seed@noreply.github.com>
Cc: Nicholas Long <Nicholas.Long@nrel.gov>, Review requested <review_requested@noreply.github.com>
Oggetto: Re: [SEED-platform/seed] Added ubid in building_syn.py for seed import (#2003)


@lainsworth8801 commented on this pull request.

________________________________

In seed/building_sync/building_sync.py<https://github.com/SEED-platform/seed/pull/2003#discussion_r342130927>:

> @@ -90,6 +90,11 @@ class BuildingSync(object):

                 "required": True,

                 "type": "string",

             },

+            "ubid": {

+                "path": "auc:Sites.auc:Site.auc:Buildings.auc:Building.auc:Ubid",

Don't believe i did. There isn't any if i remember right but I modified one locally for testing.

File: seed/building_sync/building_sync.py (hunk 1 above)
Comment (lainsworth8801) [reply seed/building_sync/building_sync.py]: ok thanks, I'll correct that.

**Pull request merged**